### PR TITLE
Style, eslint, and type strengthening

### PR DESCRIPTION
**Pull request by pihart**

### CODE REVIEW

File: src/lib/page.ts [hunk 1]
```diff
@@ -70,13 +70,13 @@ export default class Page extends fabric.Canvas {
...
-    const res = objects.map((object) => object.toObject(["strokeUniform"]));
-    reselect &&
+    if (reselect) {
+      this.discardActiveObject();
       this.setActiveObject(
         new fabric.ActiveSelection(selection, { canvas: this })
       );
-    return res;
+    }
+    return objects.map((object) => object.toObject(["strokeUniform"]));
```
Comment (cjquines) [inline src/lib/page.ts:79]: i belated realize that this doesnt match local style of using `obj` rather than `object`

File: src/lib/fabric.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { fabric } from "fabric";
```
Comment (cjquines) [inline src/lib/fabric.ts:1]: i feel like morally we should be PRing to DefinitelyTyped/DefinitelyTyped

Comment (pihart) [reply src/lib/fabric.ts:1]: We should but any changes made there should be definitively correct and I certainly don't know that yet

**Comment by pihart** [comment]
Is `fabric.Object().id` even a real thing? I don't see it in the fabric source or docs anywhere

**Comment by cjquines** [comment]
lgtm. should i merge?

**Pull request merged**